CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 0.62.0
+
+- Version and revision are no more updated in `SPEC` files when the `--no-bump` flag is specified and `None` revisions are no more created. (#1753)
```

Comment:
Revision is release? How about this?

- Fixed an issue with version and release being updated even if `--no-bump` flag was specified. Also fixed an issue when `None` appeared in release instead of a number.

Reply:
Hm, if I understand https://github.com/packit/packit/issues/1695 correctly, then `None` used to appear instead of an empty string (rather than a number). But otherwise this wording is more clear.

Reply:
> Hm, if I understand #1695 correctly, then `None` used to appear instead of an empty string (rather than a number). But otherwise this wording is more clear.

There were multiple errors in the code, it appears instead of the revision numbers.